templates/navigation/menu--group-menu.html.twig

```diff
@@ -15,17 +15,14 @@
  *     menu tree.
  *   - is_collapsed: TRUE if the link has children within the current menu tree
  *     that are not currently visible.
- *   - in_active_trail: TRUE if the link is in the active trail.
```

Comment:
This is just a comment saying that the variable is there and what it does, probably shouldn't remove.

Reply:
Noted and added back in, thanks for catching that.